src/streams.ts

```diff
@@ -44,6 +44,29 @@ export function waitForAll(streams: NodeJS.ReadableStream[]):
 }
 
 /**
+ * Returns the string contents of a Vinyl File object, waiting for
+ * all chunks if the File is a stream.
+ */
+export async function getFileContents(file: File): Promise<string> {
+  if (file.isBuffer()) {
+    return file.contents.toString('utf-8');
+  } else if (file.isStream()) {
```

Comment:
Yikes I don't think we check for this anywhere else.  I added personal TODO to update all the file contents-based code to use `getFileContents`